ftprime/argrecorder.py

```diff
@@ -345,9 +453,10 @@ def tree_sequence(self, samples=None):
         self.update_times()
         if self.timings is not None:
             start = timer.process_time()
-        msprime.sort_tables(nodes=self.nodes, edges=self.edges,
-                            sites=self.sites, mutations=self.mutations)
+        msprime.sort_tables(nodes=self.__nodes, edges=self.__edges,
+                            sites=self.__sites, mutations=self.__mutations)
         #                   migrations=self.migrations)
+        self.last_sorted_edge = self.edges.num_rows
```

Comment:
you'd missed this out - maaaybe this caused the error?

Reply:
sadly no